plugin/pkg/scheduler/algorithm/priorities/priorities_test.go

```diff
@@ -941,10 +941,10 @@ func TestPrioritiesRegistered(t *testing.T) {
 		if err != nil {
 			switch err.Error() {
 			case "exit status 2":
-				t.Errorf("unexpected error when checking %s", function.Name)
+				t.Errorf("unexpected error when checking %s", function.Name.Name)
```

Comment:
Actually no - I'm logging function.Name.Name on purpose to have more detailed information.